translations/base-fr.yaml

```diff
@@ -289,13 +289,13 @@ ingame:
         createMarker: Créer une balise
         delete: Supprimer
         pasteLastBlueprint: Copier le dernier patron
-        lockBeltDirection: Utiliser le plannificateur de convoyeurs
-        plannerSwitchSide: Échanger la direction du plannificateur
+        lockBeltDirection: Utiliser le planificateur de convoyeurs
+        plannerSwitchSide: Inverser la direction du planificateur
         cutSelection: Couper
         copySelection: Copier
         clearSelection: Effacer la sélection
         pipette: Pipette
-        switchLayers: Échanger les calques
+        switchLayers: Changer de calque
```

Comment:
"Layer" here would be better translated as "couche" but that evokes diapers. Rethink after wires update?

Reply:
Well, I chose "calque" because it's the word used as translation of "layer" in most software using this functionality. The literal translation isn't always the best, especially in the case of "couche"/"diapers" :-)

BTW, I think Échanger is better than Changer, because to switch != to change in english. The semantic isn't the same. And I'm not even talking of "Changer de couche"... "Basculer le calque" would be even better I think.

Reply:
But I'm probably a little bit too pedantic (and maybe even wrong), your translations are very good !

Reply:
Re: "layer": I'm aware of "calque" used in image editing software, but here the metaphor is more of literal layers of terrain, with underground wires. "Niveau" would work if it wasn't [already used for game levels](https://www.giantitp.com/comics/oots0012.html). Had a look at what SimCity uses for the water layer and it doesn't talk about layers at all, just "vue"/"affichage"/"carte".

Re: "switch": To me "échanger" suggests it affects the game state (somehow swaps the layers), not just the interface. "Basculer" is very good, thanks. But it will need to be changed if there are ever more than 2 layers.

Reply:
You're right. At the time, the intended game semantic wasn't even properly specified (is it now ?). I quite sure translations would need to be adjusted further in the development line.